src/qibotn/backends/quimb.py

```diff
@@ -16,9 +16,11 @@ def __init__(self, runcard):
 
             mps_enabled_value = runcard.get("MPS_enabled")
             if mps_enabled_value is True:
-                self.MPS_enabled = True
+                self.mps_opts = {"method": "svd", "cutoff": 1e-6, "cutoff_mod": "abs"}
             elif mps_enabled_value is False:
-                self.MPS_enabled = False
+                self.mps_opts = False
```

Comment:
```suggestion
                self.mps_opts = None
```
In general, use booleans together (`True` and `False`) and use `None` to indicate the absence of some value.

Reply:
Sure, it is changed.